gatsby-node.js

```diff
@@ -685,8 +685,16 @@ exports.sourceNodes = async ({
 }) => {
   const { createNode } = actions
 
-  const data = [{ title: "Docs", path: "/" }]
-  if (glob.sync(`${process.env.LOCAL_PROJECT_DIRECTORY}/blog/**`).length > 0) {
+  const blogFiles = glob.sync(`${process.env.LOCAL_PROJECT_DIRECTORY}/blog/**`)
```

Comment:
do we have a list of acceptable blog file extensions (e.g.: `[.md .html ...]`)? or are we really ok with all-the-things?

Reply:
This is a valid comment. I think with the current implementation you could put any file, say an image, and we'd render a site with the blog even if there isn't anything to render.

I believe Gatsby has a set of allowed extensions that make sense for a blog. However, I'd push toward having our own subset. I believe sticking with markdown for the moment is a better move. 

What do you think?

Reply:
yeah, IMO markdown & any supporting file types for the time being would be better (so as you said, a subset of the gatsby defaults). we could then monitor use-cases coming in from users, build up the list based on the 90% of cases.

Reply:
Solid point folks. I've switched it to only search for markdown in the blog posts folder for now.